pydicom/values.py

```diff
@@ -463,17 +463,14 @@ def convert_PN(
 
     Returns
     -------
-    valuerep.PersonName or MultiValue of PersonName
-        The decoded 'PN' value(s).
+    valuerep.PersonName
+        The decoded 'PN' value.
     """
-    def get_valtype(x: bytes) -> PersonName:
-        return PersonName(x, encodings).decode()
 
-    b_split = byte_string.rstrip(b'\x00 ').split(b'\\')
-    if len(b_split) == 1:
-        return get_valtype(b_split[0])
-
-    return MultiValue(get_valtype, b_split)
+    encodings = encodings or [default_encoding]
+    stripped_string = byte_string.rstrip(b'\x00 ')
+    decoded_value = decode_bytes(stripped_string, encodings, TEXT_VR_DELIMS)
+    return PersonName(decoded_value, encodings, stripped_string).decode()
```

Comment:
I think this is not correct, as it does not handle multi-value PN. The bug was that the split was done on the encoded byte values instead of the (decoded) string values, not that multi-value PN is not possible. But maybe I'm missing something - I will have a closer look tonight (I had only a cursory glance yesterday, as I wasn't on a PC).

Reply:
According [this reference](https://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_6.2.html):

> The character code 5CH (the BACKSLASH "\" in ISO-IR 6) shall not be present, as it is used as the delimiter between Values in multi-valued Data Elements.

So my understanding is that PN can only be single valued.

Reply:
> So my understanding is that PN can only be single valued.

On the contrary: the backslash is _only_ allowed in VRs that are not multi-valued. In VRs that _can_ be multi-value, it cannot appear _inside a single value_, because it will always interpreted as a value delimiter.

Reply:
There are multiple PN tags that can be multi-valued (for example `NameOfPhysiciansReadingStudy`).

Reply:
Thanks @mrbean-bremen , please let me fix this :)

Reply:
Hope it's better now :pray: